pubtools/_pulp/tasks/set_maintenance/base.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+from pubtools._pulp.task import PulpTask
+from pubtools._pulp.services import PulpClientService
+
+
+step = PulpTask.step
+LOG = logging.getLogger("set-maintenance")
+
+
+class SetMaintenance(PulpClientService, PulpTask):
+    def add_args(self):
+        super(SetMaintenance, self).add_args()
+
+        self.parser.add_argument("--owner", help="who sets/unsets maintenance mode")
+
+        self.parser.add_argument(
+            "--repo-url-regex",
+            help="set repositories whose relative_url matches to this regex to maintenance mode",
```

Comment:
This is a base class but this help message only seems true for the SetMaintenanceOn case.  At least, my reading of this text is that it means maintenance mode will be *enabled* for the matched repos.

Also, as a minor thing I think it's better to have a user-oriented term like "publish URL" than the implementation detail of a field named relative_url.


Would this be better?

```suggestion
            help="adjust maintenance mode for repositories with a publish URL matching this pattern",
```

Reply:
yeah, this is better.